app/models/skin.rb

```diff
@@ -3,14 +3,14 @@
 include CssCleaner
 
 class Skin < ActiveRecord::Base
-  
+
   TYPE_OPTIONS = [
                    [ts("Site Skin"), "Skin"],
                    [ts("Work Skin"), "WorkSkin"],
                  ]
-    
+
   # any media types that are not a single alphanumeric word have to be specially handled in get_media_for_filename/parse_media_from_filename
-  MEDIA = %w(all screen handheld speech print braille embossed projection tty tv) + ['only screen and (max-width: 640px)']
+  MEDIA = %w(all screen handheld speech print braille embossed projection tty tv) + ['only screen and (max-width: 42em)'] + ['only screen and (max-width: 62em)']
```

Comment:
This file has a bunch of whitespace changes, so I'm playing comment to indicate the code change. Here's a codechange!